hivemind/optim/collaborative.py

```diff
@@ -101,6 +97,9 @@ class CollaborativeOptimizer(DecentralizedOptimizerBase):
     :param kwargs: additional parameters forwarded to DecentralizedAverager
     :note: If you are using CollaborativeOptimizer with lr_scheduler, it is recommended to pass this scheduler
       explicitly into this class. Otherwise, scheduler may not be synchronized between peers.
+    :note: **For new projects, please use hivemind.Optimizer**. CollaborativeOptimizer is an older version of that.
```

Comment:
I think it is important to keep that in the beginning of the corresponding docs section: https://learning-at-home.readthedocs.io/en/latest/modules/optim.html#module-hivemind.optim.collaborative

Otherwise, people are likely to miss this warning while taking a quick look on the docs.

Reply:
Moved it higher, but below the overview of the class